src/openrct2/entity/EntityRegistry.cpp

```diff
@@ -19,6 +19,7 @@
 #include "../entity/Staff.h"
 #include "../interface/Viewport.h"
 #include "../peep/RideUseSystem.h"
+#include "../profiling/Profiling.h"
```

Comment:
You are including the header here, but this file does not seem to have any profiled functions.

Reply:
You are right, this probably slipped on the rebase.